### PR TITLE
Periodic table

### PR DESCRIPTION
**Pull request by TheDoctorRAB**
This is a boundary node for the shell-model.

### CODE REVIEW

File: periodic-table/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE
```
Comment (katyhuff) [inline periodic-table/index.md]: The title doesn't need to be in caps.

File: periodic-table/index.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+learning_objectives: 
+  - 
+references:
+    -
+abet_outcomes:
+  -  
+assessments: 
+  - 
+...
+# Periodic table
```
Comment (katyhuff) [inline periodic-table/index.md]: The title will be published with the web interface if the "layout: node" metadata is included above.

File: periodic-table/index.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+## Learning objective
```
Comment (katyhuff) [inline periodic-table/index.md]: The learning objectives will automatically be put on top in the web interface if the "layout: node" metadata is included above.

File: periodic-table/index.md (hunk 1 above)
Comment (katyhuff) [inline periodic-table/index.md]: first item, before title, should be layout: node

File: periodic-table/index.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  - 
+learning_objectives: 
+  -
```
Comment (katyhuff) [inline periodic-table/index.md:7]: do boundary nodes need learning objectives?

File: periodic-table/index.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  -
```
Comment (katyhuff) [inline periodic-table/index.md]: I think you can just leave this out or put None

File: periodic-table/index.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
+---
+title: PERIODIC TABLE 
+uuid: periodic-table
+prerequisites:
+  - 
+learning_objectives: 
+  - 
+references:
```
Comment (katyhuff) [inline periodic-table/index.md:8]: This blank item will throw an error. Try "references: None" or just leave references out of the header entirely.

File: periodic-table/index.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
...
+    -
+abet_outcomes:
+  -
```
Comment (katyhuff) [inline periodic-table/index.md]: This blank item will throw an error. Try "references: None" or just leave abet out of the header entirely.

File: periodic-table/index.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+prerequisites:
+  - 
+learning_objectives: 
+  - 
+references:
+    -
+abet_outcomes:
+  -  
+assessments: 
+  -
```
Comment (katyhuff) [inline periodic-table/index.md]: This blank item will throw an error. Try "references: None" or just leave abet out of the header entirely.

**Review by katyhuff — CHANGES_REQUESTED** [review]
Thanks @TheDoctorRAB ! This is a great example of a boundary node. Just a couple of syntax issues.

File: periodic-table/index.md (hunk 4 above)
Comment (TheDoctorRAB) [reply periodic-table/index.md:7]: I left it like that because I figured in the future someone might want to come in and populate the node.

**Comment by TheDoctorRAB** [comment]
I'll modify it later.

**Comment by TheDoctorRAB** [comment]
This was meant to be a boundary node for the shell-model. We can close it.

**Comment by katyhuff** [comment]
for now, no boundary nodes.

**Pull request closed without merging**